GCRCatalogs/cosmodc2.py

```diff
@@ -451,6 +451,13 @@ def _generate_quantity_modifiers(self):
 
         return quantity_modifiers
 
+    def get_quantity_info(self,q):
+        info_dict = self._get_quantity_info_dict(q)
+        if q == 'hostHaloMass':
+            if 'units' in info_dict:
+                info_dict['units'] = 'Msun/h'
+        
+        return info_dict
```

Comment:
We should update the upstream method instead. 

```python
    def _collect_native_quantities(self, fh, collect_info_dict=False):
        orig_output = super()._collect_native_quantities(fh, collect_info_dict)
        # a hot fix of the unit of native halo mass (hostHaloMass), which should be Msun/h
        if collect_info_dict:
            orig_output[1]['hostHaloMass']['unit'] = 'Msun/h'
        return output
```

Reply:
OK, this is now implemented and tested. This PR is ready to merge. baseDC2 does not have units so no modifications are necessary. protoDC2 remains to be checked.